<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddChat</class>
 <widget class="QDialog" name="AddChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/src/img/65456.png</normaloff>:/src/img/65456.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5683A4;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color:#519BC7;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_create">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color:#5683A4;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QWidget" name="loginForm" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>571</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#loginForm{background-color:white;
border-radius:10px;
}</string>
   </property>
   <widget class="QLabel" name="dialog_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font-weight:bold;
font-size:18px;
background-color:white;</string>
    </property>
    <property name="text">
     <string>creating</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;
font-weight:400;
font-size:14px;
background-color:white;</string>
    </property>
    <property name="text">
     <string>Please fill infomation</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
    </property>
    <property name="text">
     <string>....</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_chat_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#input_chat_name
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color: white;
}
#input_chat_name:focus
{
border-bottom:1px solid #40ABE3;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
    </property>
    <property name="text">
     <string>....</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_chat_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#input_chat_title
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color: white;
}
#input_chat_name_2:focus
{
border-bottom:1px solid #40ABE3;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dst">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>loginForm</zorder>
  <zorder>btn_create</zorder>
  <zorder>btn_cancel</zorder>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
